CMakeLists.txt

```diff
@@ -58,6 +58,10 @@ option(SWIFT_INCLUDE_DOCS
     "Create targets for building docs."
     TRUE)
 
+option(SWIFT_ANALYZE_CODE_COVERAGE
```

Comment:
However, we still won't be able to revert to `if(SWIFT_ANALYZE_CODE_COVERAGE)` because it'll default to `FALSE` for all the possible cases.